src/app/pages/product-item.page/product-routing.module.ts

```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { ProductItemPageComponent } from './product-item.page.component';
+
+export const routes: Routes = [
+  { path: '', component: ProductItemPageComponent }, // /products/ route
+  { path: ':id', component: ProductItemPageComponent }, // /products/:id route
```

Comment:
Are you sure is it not `'/:id'`?

Reply:
Yes, I tried with '/:id', and it didnt work. I had this URL: /products//1